File: PYTHON/Baju.py
from Aksesoris import Aksesoris

class Baju(Aksesoris):
    size = ""
    merk = ""
    
    def __init__(self, idProduk=0, nama="", harga=0, stok=0, jenis="", bahan="", warna="", size="", merk=""):
        super().__init__(idProduk, nama, harga, stok, jenis, bahan, warna)  # Memanggil konstruktor Aksesoris
        self.size = size
        self.merk = merk

    def setSize(self, size): self.size = size
    def getSize(self): return self.size    

    def setMerk(self, merk): self.merk = merk
    def getMerk(self): return self.merk

File: JAVA/Baju.java
class Baju extends Aksesoris
{
  private String size;
  private String merk;

  // konstruktor
  public Baju(){

  }

  public Baju(int id, String nama, int harga, int stok, String jenis, String bahan, String warna, String size, String merk){
    super(id, nama, harga, stok, jenis, bahan, warna);
    this.size = size;
    this.merk = merk;
  }

  // getter & setter
  public String getSize(){
    return size;
  }

  public void setSize(String size){
    this.size = size;
  }

  public String getMerk(){
    return merk;
  }

  public void setMerk(String merk){
    this.merk = merk;
  }
}